blog/2019-10-02-MigratingOptaWebEmployeeRosteringToSpringBoot.adoc

```diff
@@ -0,0 +1,195 @@
+= Migrating OptaWeb Employee Rostering Backend Application to Spring Boot
+:page-interpolate: true
+:awestruct-author: jucui
+:awestruct-layout: blogPostBase
+:awestruct-tags: [employee rostering, coding, algorithm, production]
+
+Recently, the OptaPlanner team is shifting towards cloud-native applications.
```

Comment:
I'm not a native speaker, but I believe the correct tense after `recently` has to be `has been shifting`.

Reply:
Very good catch! I agree, present perfect tense works better here.